test/indexer/test_hca_indexer.py

```diff
@@ -844,7 +861,7 @@ def test_cell_line_sample(self):
                     cell_lines_model_organ = {cl['model_organ'] for cl in contents['cell_lines']}
                 self.assertEqual(cell_lines_model_organ, {'blood (parent_cell_line)', 'blood (child_cell_line)'})
                 self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (child_cell_line)'])
-                self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [config.null_keyword])
+                self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [Document.translate_field(None, str)])
```

Comment:
Might make sense to extract this into a variable like you do above even though it may only be used once.